Mathlib/Util/Qq.lean

```diff
@@ -35,4 +35,16 @@ def findLocalDeclWithTypeQ? {u : Level} (sort : Q(Sort u)) : MetaM (Option Q($so
   let some fvarId ← findLocalDeclWithType? q($sort) | return none
   return some <| .fvar fvarId
 
+/-- Returns a proof of `p : Prop` using `decide p`.
+
+This is a Qq version of `Lean.Meta.mkDecideProof`. -/
+def mkDecideProofQq (p : Q(Prop)) : MetaM Q($p) := mkDecideProof p
```

Comment:
This is no longer used, but I'd be happy to merge it in another PR (if you also eliminated existing callers of `mkDecideProof`)

Reply:
Will do